test/remote/password_reset_tests.js

```diff
@@ -36,6 +36,7 @@ test(
       function (err, req, res, obj) {
         t.notOk(err, 'good request is successful')
         t.equal(res.statusCode, 200, 'good request returns a 200')
+        t.ok(obj, 'got an obj, make jshint happy')
```

Comment:
After about the 40th time of reading 'make jshint happy', I'm starting to lean towards "relax the option, this is silly".

Maybe change .jshintrc's `unused` to ["vars"](https://jslinterrors.com/unused-a) (per [mozilla/fxa-auth-server](https://github.com/mozilla/fxa-auth-server/blob/master/.jshintrc#L8)). Or counter proposal of creating a test/.jshintrc with that rule disabled.

> `vars` &mdash; Only warn about unused variable and function declarations. Don't warn about any function parameters.
> &mdash; via https://jslinterrors.com/unused-a

---

> This can be set to `vars` to only check for variables, not function parameters, or `strict` to check all variables and parameters. The default (`true`) behavior is to allow unused parameters that are followed by a used parameter.
> &mdash; via http://jshint.com/docs/options/#unused